css-four-column-layout/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS challenge - Four Column Layout</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container padding bg-color">
+      <div class="row wrap center">
+        <div class="column-full">
+          <h4>How it Works</h4>
+        </div>
+        <div class="column-fourth">
+          <img class="img-150px" src="images/screen-1.png" alt="App settings screen">
+        </div>
+        <div class="column-fourth">
+          <img class="img-150px" src="images/screen-2.png" alt="Uber navigation screen - trip ignored">
+        </div>
+        <div class="column-fourth">
+          <img class="img-150px" src="images/screen-3.png" alt="Lyft settings screen - auto accept">
+        </div>
+        <div class="column-fourth">
+          <img class="img-150px" src="images/screen-4.png" alt="Uber settings screen - Stats">
+        </div>
+        <div class="column-fourth">
+          <img class="img-40px margin-top-neg5" src="images/switch-icon.png" alt="Switch icon">
+          <h6 class="margin-tb">App Switching</h6>
+          <p>Toggles all your apps on and off.</p>
+        </div>
+        <div class="column-fourth">
+          <img class="img-40px margin-top-neg5" src="images/filter-icon.png" alt="Filter icon">
+          <h6 class="margin-tb">Trip Filtering</h6>
+          <p>Evaluates your trip requests.</p>
+        </div>
+        <div class="column-fourth">
+          <img class="img-40px margin-top-neg5" src="images/slider-icon.png" alt="Slider icon">
+          <h6 class="margin-tb">Service Filters</h6>
+          <p>Accepts only your ideal trips.</p>
+        </div>
+        <div class="column-fourth">
+          <img class="img-40px margin-top-neg5" src="images/bars-icon.png" alt="Bars Icon">
+          <h6 class="margin-tb">Stats + Mileage</h6>
+          <p>Tracks your trips across all apps.</p>
+        </div>
+      </div>
```

Comment:
I can see why you would do this from the perspective of layout classes, but this organization makes it seem like the images are not related to the icons and text. You should have the `img` in the same column as the rest of the content for each column.

Reply:
ah gotcha, I thought I was keeping everything related in one row. I will from now on keep related content all in the same column!